### PR TITLE
Asp net

### PR DESCRIPTION
**Pull request by standridgejeff**
Ready for review

### CODE REVIEW

**Comment by standridgejeff** [comment]
Made requested Changes

File: AspNetCoreMvc/Controllers/ProductApiController.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        // GET: api/ProductApi
+        [HttpGet]
+        public async Task<IEnumerable<Product>> Get()
+        {
+            return await _prodRepo.GetProducts();   
+        }
+
+        // GET: api/ProductApi/5
+        [HttpGet("{id}", Name = "Get")]
+        public Product Get(int id)
```
[the remark this answers is not in the document]
Comment (codytwinton) [reply AspNetCoreMvc/Controllers/ProductApiController.cs]: Still needed.

File: ITB.Repositories/ProductRepository.cs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        public int UpdateProduct(Product prod)
+        {
+            using (var conn = _conn)
+            {
+                conn.Open();
+                return conn.Execute("UPDATE product SET Name = @Name WHERE ProductId = @Id", prod);
+            }
+        }
+
+        public Product GetProduct(int Id)
```
[the remark this answers is not in the document]
Comment (codytwinton) [reply ITB.Repositories/ProductRepository.cs]: Still needed.

File: ITB.Shared/IProductRepository.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace ITB.Shared
+{
+    public interface IProductRepository
+    {
+        Task<IEnumerable<Product>> GetProducts();
+        Product GetProduct(int Id);
```
[the remark this answers is not in the document]
Comment (codytwinton) [reply ITB.Shared/IProductRepository.cs]: Still needed.

**Pull request merged**